_rules/html-page-title-descriptive-c4a8a4.md

```diff
@@ -175,4 +172,22 @@ This `title` element is a child of an `svg` element.
 </svg>
 ```
 
-[html]: #namespaced-element
+#### Failed Example 3
```

Comment:
This example is now inapplicable based on comments that empty titles are catched by [HTML page has non-empty title](https://github.com/act-rules/act-rules.github.io/pull/1912/files/#:~:text=HTML%20page%20has%20non%2Dempty%20title)